inference-engine/src/cldnn_engine/cldnn_infer_request.h

```diff
@@ -46,6 +46,10 @@ class CLDNNInferRequest : public InferenceEngine::InferRequestInternal {
     void EnableProfiling() { m_useProfiling = true; }
     void EnableStreams() { m_useStreams = true; }
 
+    InferenceEngine::StatusCode Cancel() override {
+        return InferenceEngine::NOT_IMPLEMENTED;
+    }
```

Comment:
I suggest putting this code to the base class as default fallback implementation to avoid modification of all plugins.

Reply:
I  had this idea, but wasn't sure if it will be good to implement a method at the interface. But agree, this will help to avoid lot  of changes in plugins with no support for  cancellation.

Reply:
It is not an interface class, it is internal base class for implementations. So I think it is OK to have there default fallback implementations.

Reply:
Added default implementation, did remove Cancel implementation from  all plugins except CPU